src/current/v23.2/cockroach-sql-binary.md

```diff
@@ -23,66 +23,20 @@ The output of `cockroach-sql` when used non-interactively is part of a stable in
     <button class="filter-button page-level" data-scope="windows"><strong>Windows</strong></button>
 </div>
 
-Download the binary and copy it into your `PATH`.
+1. Visit [Releases]({% link releases/index.md %}) and download the SQL Shell binary for CockroachDB.
 
 <section class="filter-content" markdown="1" data-scope="linux">
-
-{% include_cached copy-clipboard.html %}
-~~~ shell
-curl https://binaries.cockroachdb.com/cockroach-sql-{{ page.release_info.version }}.linux-amd64.tgz | tar -xz && sudo cp -i cockroach-sql-{{ page.release_info.version }}.linux-amd64/cockroach-sql /usr/local/bin/ && if [ ! -f /usr/local/bin/cockroach ]; then sudo ln -s /usr/local/bin/cockroach-sql /usr/local/bin/cockroach; fi
-~~~
-
-If you don't have an existing `cockroach` binary in `/usr/local/bin` this will create a symbolic link to `cockroach` so you can use the `cockroach sql` command.
-
+1. Follow the instructions to [install CockroachDB]({% link {{ page.version.version }}/install-cockroachdb.md %}) on your local system. The resulting `cockroach` binary supports only `cockroach sql` subcommands.
```

Comment:
the JEWEL of this PR

now if only we could get the Powershell script out of the windows install (but that's another battle for another day)